### PR TITLE
Enable to check if etcd is on path in start_simulator.sh

### PR DESCRIPTION
**Pull request by sanposhiho**
Hello team.

This PR enable to check if etcd is on path in start_simulator.sh.

### CODE REVIEW

File: hack/start_simulator.sh [hunk 1]
```diff
@@ -29,8 +29,10 @@ cleanup_etcd() {
   echo "Clean up finished"
 }
 
-trap cleanup_etcd EXIT
 
+checkEtcdOnPath
+
+trap cleanup_etcd EXIT
```
Comment (adtac) [inline hack/start_simulator.sh:35]: trap after start_etcd please

Comment (sanposhiho) [reply hack/start_simulator.sh]: fixed thanks

File: hack/start_simulator.sh [hunk 2]
```diff
@@ -29,9 +29,11 @@ cleanup_etcd() {
   echo "Clean up finished"
 }
 
-trap cleanup_etcd EXIT
+
+checkEtcdOnPath
```
Comment (haosdent) [inline hack/start_simulator.sh:33]: This fun name looks weird. How about use checkIfEtcdExists

File: hack/start_simulator.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@ checkEtcdOnPath() {
   echo "Cannot find etcd on PATH."
```
Comment (haosdent) [inline hack/start_simulator.sh:8]: PATH -> $PATH

File: hack/start_simulator.sh (hunk 2 above)
Comment (haosdent) [reply hack/start_simulator.sh:33]: Better to use check_if_etcd_exists to avoid we mix snake style and camel style in this file.

**Comment by sanposhiho** [comment]
@haosdent 
Thanks for reviewing. I fixed some wording as your suggestions.

**Comment by haosdent** [comment]
you are welcome @sanposhiho

**Comment by 196Ikuchil** [comment]
I ran `./hack/start_simulator.sh` in my local, and verified that the messages are printed if `which etcd` command fails (etcd is not installed).

/lgtm

**Pull request merged**